.dockerignore

```diff
@@ -40,9 +40,6 @@
 !scripts/in_container
 !scripts/docker
 
-# Add provider packages to the context
-!provider_packages
```

Comment:
Did you mean to include this change here?

Reply:
I can easily separate it out. It's also optimizing the image build experience, but I can move it to separate PR.

Reply:
Done.